templates/firefox-ios.xliff

```diff
@@ -2150,7 +2150,7 @@
       </trans-unit>
     </body>
   </file>
-  <file original="Client/Menu.strings" source-language="en" datatype="plaintext">
+  <file original="Shared/Supporting Files/Menu.strings" source-language="en" datatype="plaintext">
```

Comment:
This group of strings will move around in the string export, since this element is not in the right place within the file, but we shouldn't be losing translations.